### PR TITLE
Update rookout chart environment variables

### PR DESCRIPTION
**Pull request by Zohar2506**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rookout/Chart.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: "0.2.3"
```
Comment (davidkarlsen) [inline stable/rookout/Chart.yaml:2]:
```suggestion
appVersion: "0.3.9"
```
which is the image tag?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by Zohar2506** [comment]
/davidkarlsen what else needs to be done to merge it?

**Comment by Zohar2506** [comment]
https://github.com/helm/charts/pull/11306

**Pull request closed without merging**